client/header/activity-panel/panels/inbox/abbreviated-card.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { Card, CardBody, Icon } from '@wordpress/components';
+import { Text } from '@woocommerce/experimental';
+import { Link } from '@woocommerce/components';
+import { recordEvent } from '@woocommerce/tracks';
+
+export const AbbreviatedCard = ( {
```

Comment:
Can this be an exported/public component? I think design has been looking at our UI elements as "available building blocks" for future work, but they need part of our published packages to be usable by 3PDs and other teams.

Reply:
I like the idea. I moved the component in the commit 6578cc3